arches/app/views/api.py

```diff
@@ -455,8 +455,10 @@ def get(self, request, nodeid, zoom, x, y):
                         cursor.execute(
                             """SELECT ST_AsMVT(tile, %s, 4096, 'geom', 'id') FROM (SELECT tileid,
                                 id,
+                                featureid AS tile_featureid,
                                 resourceinstanceid,
                                 nodeid,
+                                featureid::text AS tile_featureid,
```

Comment:
you seem to have the column duplicated in this select statement.  Also, is there a reason to use the name `tile_featureid` over the simpler and more consistent `featureid`?

Reply:
change in latest